src/ILVerify/src/Program.cs

```diff
@@ -97,169 +85,146 @@ private ArgumentSyntax ParseCommandLine(string[] args)
-                        {
-                            message.Append(parameter.ToString());
-                            message.Append(", ");
-                        }
-                        message.Remove(message.Length - 2, 2);
-                    }
-                    message.Append(")");
-                    message.Append("]");
+                Help(syntax.GetHelpText());
+                return 1;
+            }
 
-                    message.Append("[offset 0x");
-                    message.Append(args.Offset.ToString("X8"));
-                    message.Append("]");
+            if (_inputFilePaths.Count == 0)
+                throw new CommandLineException("No input files specified");
 
-                    if (args.Found != null)
-                    {
-                        message.Append("[found ");
-                        message.Append(args.Found);
-                        message.Append("]");
-                    }
+            _verifier = new Verifier(this);
+            _verifier.SetSystemModuleName(_systemModule);
 
-                    if (args.Expected != null)
-                    {
-                        message.Append("[expected ");
-                        message.Append(args.Expected);
-                        message.Append("]");
-                    }
+            foreach (var kvp in _inputFilePaths)
+            {
+                var results = VerifyAssembly(new AssemblyName(kvp.Key));
 
-                    if (args.Token != 0)
-                    {
-                        message.Append("[token  0x");
-                        message.Append(args.Token.ToString("X8"));
-                        message.Append("]");
-                    }
+                foreach (var result in results)
+                {
+                    PrintResult(result, kvp.Value);
+                }
 
-                    message.Append(" ");
+                var numErrors = results.Count();
+                if (numErrors > 0)
+                    WriteLine(numErrors + " Error(s) Verifying " + kvp.Value);
+                else
+                    WriteLine("All Classes and Methods in " + kvp.Value + " Verified.");
+            }
 
-                    if (_stringResourceManager == null)
-                    {
-                        _stringResourceManager = new ResourceManager("ILVerify.Resources.Strings", Assembly.GetExecutingAssembly());
-                    }
+            return 0;
+        }
 
-                    var str = _stringResourceManager.GetString(args.Code.ToString(), CultureInfo.InvariantCulture);
-                    message.Append(string.IsNullOrEmpty(str) ? args.Code.ToString() : str);
+        private void PrintResult(VerificationResult result, string pathOrModuleName)
+        {
+            Write("[IL]: Error: ");
+
+            Write("[");
+            Write(pathOrModuleName);
+            Write(" : ");
+            Write(result.TypeName);
+            Write("::");
+            Write(result.Method);
```

Comment:
I do not think this will print a nice name of the method.

Reply:
Thanks
Fixed this and verified manually. Here's an example of output:
```
[IL]: Error: [D:\issues\Console46\Console46\bin\Debug\New.exe : Program::set_Property([mscorlib]System.Int32)][offset 0x00000002] Fall through end of the method without returning.
1 Error(s) Verifying D:\issues\Console46\Console46\bin\Debug\New.exe
```